Marlin/Marlin_main.cpp

```diff
@@ -3967,18 +3967,16 @@ inline void gcode_M303() {
 }
 
 #ifdef SCARA
-
   /**
    * M360: SCARA calibration: Move to cal-position ThetaA (0 deg calibration)
```

Comment:
I like this kind of PR. Edit the comment on this line, and it will be ready to go!